autonomi/Cargo.toml

```diff
@@ -26,8 +26,9 @@ extension-module = ["pyo3/extension-module", "pyo3-async-runtimes"]
 loud = []
 
 [dependencies]
-ant-bootstrap = { path = "../ant-bootstrap", version = "0.2.1" }
+ant-bootstrap = { path = "../ant-bootstrap", version = "0.2.2" }
 ant-evm = { path = "../ant-evm", version = "0.1.13" }
+ant-networking = { path = "../ant-networking", version = "0.3.11" }
```

Comment:
We shouldn't be needing _ant-networking_ here!

Reply:
Yeeted: https://github.com/maidsafe/autonomi/pull/2955/commits/da01124d9750b009619ae00889cf0ef8b5c34dd2